### PR TITLE
[PDI-20109] - Extra object of LoggingObject created on execution of Transformation

### PR DESCRIPTION
**Pull request by rmansoor**

### CODE REVIEW

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :warning: Build finished in 22m 48s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl engine
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 7 test(s) failed:

<details><summary><b>org.pentaho.di.trans.steps.datagrid.DataGrid_EmptyStringVsNull_Test.emptyAndNullsAreNotDifferent</b> (click to expand)</summary><p>

```
[1][0] expected:<null> but was:<>
```

</p></details>

<details><summary><b>org.pentaho.di.trans.steps.fieldsplitter.FieldSplitterTest.testSplitFields</b> (click to expand)</summary><p>

```
Output row is of an unexpected length expected:<4> but was:<3>
```

</p></details>

<details><summary><b>org.pentaho.di.trans.steps.fieldsplitter.FieldSplitter_EmptyStringVsNull_Test.emptyAndNullsAreNotDifferent</b> (click to expand)</summary><p>

```
[1][1] expected:<null> but was:<>
```

</p></details>

<details><summary><b>org.pentaho.di.trans.steps.fileinput.text.TextFileInputTest.fieldsWithLineBreaksWithEmptyLinesTest</b> (click to expand)</summary><p>

```
[0][0] expected:<null> but was:<>
```

</p></details>

<details><summary><b>org.pentaho.di.trans.steps.fileinput.text.TextFileInputTest.readInputWithNonEmptyNullif</b> (click to expand)</summary><p>

```
expected null, but was:<>
```

</p></details>

<details><summary><b>org.pentaho.di.trans.steps.fileinput.text.TextFileInputTest.readInputWithMissedValues</b> (click to expand)</summary><p>

```
[0][1] expected:<[1]> but was:<[]>
```

</p></details>

<details><summary><b>org.pentaho.di.trans.steps.textfileinput.TextFileInputTest.readInputWithMissedValues</b> (click to expand)</summary><p>

```
[0][1] expected:<[1]> but was:<[]>
```

</p></details>

  
Tests run: **2325**, Failures: **7**, Skipped: **13** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-9357/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: engine/src/main/java/org/pentaho/di/job/entries/trans/JobEntryTrans.java [hunk 1]
```diff
@@ -1135,8 +1135,7 @@ public Result execute( Result result, int nr ) throws KettleException {
           //
           //trans = new Trans( transMeta, this );
```
Comment (ddiroma) [inline engine/src/main/java/org/pentaho/di/job/entries/trans/JobEntryTrans.java:1136]: @rmansoor please update copyright header

Comment (rmansoor) [reply engine/src/main/java/org/pentaho/di/job/entries/trans/JobEntryTrans.java:1136]: updated the header @ddiroma

**Review by ddiroma — APPROVED** [review]

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Pull request merged**